src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -224,6 +224,7 @@ public MarkupResult ParseHtml(string html, FileAndType ft)
                     sources.Add(new LinkSourceInfo
                     {
                         Target = file,
+                        Anchor = anchor,
```

Comment:
Finally find it's introduced in [this](https://github.com/dotnet/docfx/commit/051548b6ecf6b4469a2dc56951154545ba309593) commit... This anchor is an intermediate attribute used in resolving xref, and has been removed [here](https://github.com/dotnet/docfx/commit/051548b6ecf6b4469a2dc56951154545ba309593#diff-7bc60e42b31ac047d2df1c6b05f100d5R450).